base/pkg/entry.jl

```diff
@@ -158,15 +158,14 @@ status(io::IO, pkg::AbstractString) = status(io, pkgname = pkg)
 function status(io::IO, pkg::AbstractString, ver::VersionNumber, fix::Bool)
     @printf io " - %-29s " pkg
     fix || return println(io,ver)
-    @printf io "%-19s" ver
+    @printf io "%-9s" ver
     if ispath(pkg,".git")
         prepo = GitRepo(pkg)
         try
+            print(io, string(LibGit2.head_oid(prepo))[1:8], "  ")
```

Comment:
(when you do, better if you push new commits, or rebase these, to the same branch instead of opening a brand new PR)